Update/onik_tips_17.txt

```diff
@@ -469,7 +469,7 @@ void main()
 		   NULL, " Considering how our pay comes from the public's hard-earned taxes,", Line_ContinueAfterTyping);
 	Wait (2800);
 	OutputLine(NULL, "まぁ時代の流れですかねぇ。\"」",
-		   NULL, " it's the way of the times, I suppose.", Line_WaitForInput);
+		   NULL, " it's the way of the times, I suppose.\"", Line_WaitForInput);
```

Comment:
If you look at the japanese line above it seems someone accidently added the quote there instead of here. Remove it in the japanese text please.